distributed/client.py

```diff
@@ -354,7 +354,7 @@ def __init__(self, address=None, start=True, loop=None, timeout=3,
         self.status = None
         self._pending_msg_buffer = []
         self.extensions = {}
-        self.scheduler_file = scheduler_file
+        self.scheduler_file = scheduler_fileF
```

Comment:
> Right now it's failing and I'm not quite sure why

Just in case you didn't see I thought I'd bring this to your attention again @mariusvniekerk